<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PauseButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>410</y>
     <width>106</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>380</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>106</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="CameraLayout">
    <item>
     <widget class="QLabel" name="CameraIDLabel">
      <property name="text">
       <string>Camera Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="CameraList"/>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="CameraView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>201</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="DetailsLayout">
    <item>
     <widget class="QLabel" name="DetailsLabel">
      <property name="text">
       <string>Details</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="RecordLabel">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>DataBase Screen Shot</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="FaceView"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ScreenShotButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>ScreenShot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RecordButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>310</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="TimeLayout">
    <item>
     <widget class="QLabel" name="TimeLabel">
      <property name="text">
       <string>Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="LCDNum"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="MirrorButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>231</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="TemperatureLayout">
      <item>
       <widget class="QLabel" name="TempertureLabel">
        <property name="text">
         <string>Temperature</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="TemperatureBrowser"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Covid19Layout">
      <item>
       <widget class="QLabel" name="Covid19Label">
        <property name="text">
         <string>Covid19 Result</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="Covid19Browser"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="ClassNumberLabel">
        <property name="text">
         <string>ClassNumber</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="ClassNumberBrowser"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="NameLabel">
        <property name="text">
         <string>       Name       </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="NameBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
